legend-engine-language-pure-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/to/HelperDomainGrammarComposer.java

```diff
@@ -114,7 +114,9 @@ public static String renderDerivedProperty(QualifiedProperty qualifiedProperty,
                 + PureGrammarComposerUtility.convertIdentifier(qualifiedProperty.name) + "("
                 + LazyIterate.collect(functionParameters, p -> p.accept(DEPRECATED_PureGrammarComposerCore.Builder.newInstance(transformer).withVariableInFunctionSignature().build())).makeString(",")
                 + ") {"
-                + LazyIterate.collect(qualifiedProperty.body, b -> b.accept(transformer)).makeString("\n")
+                + (qualifiedProperty.body.size() <= 1
```

Comment:
To confirm: Is null check required?

Reply:
Not required, since we expect it to be present in the grammar as seen [here](https://github.com/finos/legend-engine/blob/master/legend-engine-language-pure-grammar/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/domain/DomainParserGrammar.g4#L84)